public/stylesheets/main.css

```diff
@@ -167,7 +167,11 @@ margin-left: 15px;
     width: 60px;
   }
   body{
-    background-image: url("profilebg.jpg");
+    /*background-color: red;*/
+    background-image:url(ishaan.jpg);
```

Comment:
Do not keep your name as the background picture. It should be something different related to the webpage.

Reply:
I am so sorry,I completely forgot about that